include/boost/http_proto/serializer.hpp

```diff
@@ -147,6 +145,36 @@ class BOOST_SYMBOL_VISIBLE
 
     //--------------------------------------------
 
+    /** Create a new stream object associated with the
+        serializer.
+
+        The returned stream must not outlive its backing
+        serializer object.
+
+        Streams permit a user to supply input data to
+        the serializer using a bounded sequence of mutable
+        buffers.
```

Comment:
Best not to refer to "the user." Instead, refer to "the caller". Better yet, use a passive voice. For example:

"A stream may be used to invert the flow of control when the caller is supplying body data as a series of buffers."